realm/realm-library/proguard-rules.pro

```diff
@@ -4,3 +4,5 @@
 -keep @io.realm.internal.Keep class * { *; }
 -dontwarn javax.**
 -dontwarn io.realm.**
+-keep class io.realm.RealmCollection
```

Comment:
I am not sure I totally get this? If they are referenced by any code these interfaces should not be removed?
Is it because that GSON uses reflection for its type adapters and that somehow messes with ProGuard?